pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
Fair enough.  I'll try to first separate them out and then we can see what is best afterwards (still WIP with pulling out test suites from `test_parsers`)